crawler.py

```diff
@@ -1,10 +1,13 @@
 import json
 import os
-import time
 import random
+import time
 import traceback
+
 from selenium import webdriver
-from selenium.common.exceptions import NoAlertPresentException, NoSuchWindowException, TimeoutException
+from selenium.common.exceptions import (NoAlertPresentException,
```

Comment:
Let's configure flake8-isort so that we don't need to use multiple lines in cases like this

Reply:
"""
from selenium.common.exceptions import NoAlertPresentException
from selenium.common.exceptions import NoSuchWindowException
from selenium.common.exceptions import TimeoutException
"""
Is this the expected behaviour in this case?To achieve this a new isort cfg file will be added with all the configuration inside it.

Reply:
No sorry, to be clear, I meant something like before, where we have all the imports on the same line